connector-definition/connector-metadata.yaml

```diff
@@ -1,5 +1,15 @@
 packagingDefinition:
   type: ManagedDockerBuild
+nativeToolchainDefinition:
+  commands:
+    start:
+      type: ShellScript
+      bash: ./start.sh
+      powershell: ./start.ps1
+    watch:
+      type: ShellScript
+      bash: ./watch.sh
+      powershell: ./watch.ps1
```

Comment:
Yeah I quite like that this specifies exactly which shell it's using rather than the platform.